bin/check-nvmrc.js

```diff
@@ -2,22 +2,39 @@
 
 'use strict'
 
-var fs = require('fs')
-var path = require('path')
+const fs = require('fs')
```

Comment:
I'm least sure about the changes in this file. I think it's doing broadly the same thing but so much has changed I can't quite follow it.

To be honest, I'm not even really sure if we need this check any more.

From memory, I _think_ we introduced this when there was a lot of churn in the package lock file format between versions of node / npm.

The easiest way for us to avoid constantly checking in new versions of the lock file (or discarding the changes) was for the team to align on using the same version of node / npm.

That doesn't seem to be as much of an issue nowadays, so maybe we could relax the checks we're doing.

Reply:
Can I talk you through it tomorrow?

I've changed the `.nvmrc` file to an LTS release rather than Node.js v16.14.2 (18 Mar 2022)

I'm grabbing LTS information from `process.release.lts` rather than checking if major version numbers match

**.nvmrc**
```
lts/gallium
```

Reply:
As discussed, I've removed **check-nvmrc.js** and replaced it with:

**.npmrc**
```properties
engine-strict=true
```